src/components/chips/Chip.vue

```diff
@@ -36,17 +36,17 @@ const props = withDefaults(defineProps<Props>(), {
 });
 
 const emit = defineEmits<{
-  (event: 'click:close'): void;
-  (event: 'click'): void;
+  (e: 'click:close'): void;
+  (e: 'click', event: any): void;
```

Comment:
for the main library we probably have to do some generic typing here.

Reply:
it should be MouseEvent, but the type is kinda inconsistent in vue 2, in vue 3 it's fine